Source/CombatExtended/Compatibility/BlockerRegistry.cs

```diff
@@ -17,7 +17,7 @@ public static class BlockerRegistry
         private static bool enabledBCW = false;
         private static bool enabledPUF = false;
         private static bool enabledSZ = false;
-        private static List<Func<ProjectileCE, Vector3, Vector3, bool>> checkForCollisionBetweenCallbacks;
+        private static List<Func<ProjectileCE, Vector3, Vector3, IEnumerable<(Vector3 IntersectionPos, Action OnIntersection)>>> checkForCollisionBetweenCallbacks;
```

Comment:
This should still return a single `(Vector3, Action)`.  LinQ is slow, as is list/generator creation.  Since we only need a single value, we can filter as we go.

Reply:
Can't say if it improved performance...
With single (Vector3, Action)
![image](https://github.com/CombatExtended-Continued/CombatExtended/assets/78953324/92dc4a70-2a4f-465b-8ca8-3b6264e09151)
W/o
![image](https://github.com/CombatExtended-Continued/CombatExtended/assets/78953324/30b307f8-9b2c-45ad-ad0b-2b007e1765de)

Reply:
Done